src/main.ts

```diff
@@ -227,20 +230,18 @@ export function activate(context: vscode.ExtensionContext): ReturnType<typeof ge
             return
         }
         extension.linter.lintActiveFileIfEnabledAfterInterval()
-        if (extension.manager.getCachedContent(e.document.fileName) === undefined) {
+        const cache = extension.manager.getCachedContent(e.document.fileName)
+        if (cache === undefined) {
             return
         }
         const configuration = vscode.workspace.getConfiguration('latex-workshop')
-        const content = utils.stripCommentsAndVerbatim(e.document.getText())
-        const cache = extension.manager.getCachedContent(e.document.fileName)
-        if (cache !== undefined) {
-            cache.content = content
-        }
         if (configuration.get('intellisense.update.aggressive.enabled')) {
             if (updateCompleter) {
                 clearTimeout(updateCompleter)
             }
             updateCompleter = setTimeout(() => {
+                const content = e.document.getText()
+                cache.content = content
```

Comment:
We had better make a method to update the `content` property in the `Manager` class and call it here. The method could take a `TextDocument` object as the argument.

```ts
updateAaaaaa(document: vscode.TextDocument) {
...
}
```